recipes/tvb-framework/meta.yaml

```diff
@@ -0,0 +1,67 @@
+
+package:
+  name: tvb-framework
+  version: "1.5.4"
+
+source:
+  git_rev: 1aaf90c67450dfe535a7f76002f203218de330b4
+  git_url: https://github.com/the-virtual-brain/tvb-framework.git
+
+build:
+  noarch: python
+  number: 0
+  script: python setup.py install
+  skip: True  # [py>27]
+
+requirements:
+  build:
+    - python
+    - setuptools
+    - allensdk
+    - apscheduler
+    - BeautifulSoup4
+    - cfflib
+    - cherrypy
+    - coverage
+    - formencode
+    - genshi
+    - h5py
+    - lxml
+    - minixsv
+    - mod_pywebsocket
+    - Pillow
+    - psutil
+    - simplejson
+    - sqlalchemy  0.7.8
```

Comment:
Are these tight version dependencies actually necessary? They're what's in the `setup.py` so I guess they're right for the recipe too, but since you're the package developer too...are they actually needed to be so tight? :)

Reply:
Unfortunately, sqlalchemy-migrate is a dead package, which only works for us in this combination. In the next releases, we plan to replace it with  alembic.